### ISSUE TITLE
Preprocessing

### ISSUE THREAD

**Issue by cclough** [qubvel/segmentation_models#23]
For the preprocessing for ResNet models (switching RGB images to BGR):

- Just to confirm, the pre-trained resnet models are trained on BGR?
- Do the pretrained resnet models expect images to be zero mean & unit variance also, or not?

Another question: when fine tuning, how important is the 2 epoch freeze first? important/not important?

Thank you

**Comment by qubvel** [comment]
1) yes, resnet models trained with images in BGR
2) no, images should be in range 0-255 (models have batchnorm as the first layer)
3) it is not important to freeze, you can start train all model layer from the first epoch
